### PR TITLE
small inconsequential change to notebook

### PR DESCRIPTION
**Pull request by skendrew**
making a small test change, as requested.

### CODE REVIEW

File: tso_session/intro_TSO_data_pipeline.ipynb [hunk 1]
```diff
@@ -136,11 +136,74 @@
     "First the imports.\n"
```
Comment (ojustino) [inline tso_session/intro_TSO_data_pipeline.ipynb:136]: <p>You can ignore the first two warnings here; <code>flake8</code> isn't recognizing that the <code>%set_env</code> statements are not Python code.</p><p> </p><p>You can resolve the other warnings by either separating environment variable assignment into its own cell or by moving it after the imports in this one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (ojustino) [inline tso_session/intro_TSO_data_pipeline.ipynb:136]: <p>All imports should happen at the beginning of the notebook. If it's important to highlight that these are specific to Stage 2, you can mention so in the import cell with a comment above them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Comment (ojustino) [inline tso_session/intro_TSO_data_pipeline.ipynb:136]: <p>Please move these imports to the top of the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Comment (ojustino) [inline tso_session/intro_TSO_data_pipeline.ipynb:136]: <pre class="ql-syntax" spellcheck="false">Line #1.    import astropy.io.ascii as ascii
</pre><p>Please move this import to the top of the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>

Comment (skendrew) [reply tso_session/intro_TSO_data_pipeline.ipynb:136]: <p>done</p>

Comment (skendrew) [reply tso_session/intro_TSO_data_pipeline.ipynb:136]: <p>done</p>

Comment (skendrew) [reply tso_session/intro_TSO_data_pipeline.ipynb:136]: <p>done</p>

[the remark this answers is not in the document]
Comment (skendrew) [reply tso_session/intro_TSO_data_pipeline.ipynb:136]: <p>done</p>

**Comment by ojustino** [comment]
Hi @skendrew, thank you for this round of edits. You resolved most issues but there are still a few more.

When you finish, please remember to clear all cell outputs before committing, especially since this notebook can produce so many logging messages. You should also delete the highlighted reviewer cells I added below your imports.

## The three-point review (\*action required\*)

1. ✅/⚠️ Notebook execution: **I was able to run the notebook from top to bottom.**
    - *However,* one cell does not run because `miri_lrs_tso_100G10I_mirisim241_x1dints.fits` is missing from the downloaded `mri_lrs_output` folder. Could you make sure it's in the folder on Box?
    - I made edits that resolved warnings that were showing up on several of the plots.

2. ⚠️ Code style: **There are a still a few PEP8 violations.**
    - The tips I wrote in my first message's "__Instructions__" section should still be useful. (I moved the reviewer cells just below the imports, which should block most of `jwst`'s logging messages and make it easier to pick out the PEP8 warnings.)

3.  ✅ Notebook style

File: tso_session/intro_TSO_data_pipeline.ipynb [hunk 2]
```diff
@@ -37,13 +37,7 @@
     "6. [Final Steps: Tso3Pipeline()](#tso3pipe)<br>\n",
```
Comment (ojustino) [inline tso_session/intro_TSO_data_pipeline.ipynb:37]: <p>Here is the only cell that errors out. There may be a file missing from the Box folder that's downloaded earlier in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>

**Comment by skendrew** [comment]
can you maybe help me understand what I am doing wrong with the file retrieval that is causing the filename to be different? I followed your instructions but something is going wrong there. I have the data_url, and then when I do:

``hdu = fits.open(data_url)
hdu.info()``

I get:

``Filename: /Users/skendrew/.astropy/cache/download/url/62e9d60bfa0b1e135032abfd852efd06/contents``

**Comment by camipacifici** [comment]
Astropy fits.open puts the file in a temporary directory with a very much non human name.
If you want to preserve the filename, you can either create a link (you can find an example by Bryan [here](https://github.com/spacetelescope/jwebbinar_prep/blob/main/imaging_mode/imaging_mode_stage_1.ipynb)) or you can download and unzip a zip file so that all the files end up in the local directory (example [here](https://github.com/spacetelescope/jwebbinar_prep/blob/main/spec_analysis/jw8_redshift_fitting_solutions.ipynb) at "Get the library of spectra").
Let me know if any of these work!

**Comment by skendrew** [comment]
@camipacifici @ojustino can someone please fix these housekeeping issues for me? I need help with this, I have very limited time to spend on non-commissioning duties and SO much work, I don't have the time to go through these examples and find links and code and move files around.

**Comment by camipacifici** [comment]
No problem at all! I can take care of it.

**Pull request merged**